includes/class-magic-link.php

```diff
@@ -350,13 +348,7 @@ public static function generate_email( $user, $redirect_to = '', $subject, $mess
 			$message .= __( 'Log into your account by visiting the following URL:', 'newspack' ) . "\r\n\r\n";
 		}
 
-		// If the message contains a magic link placeholder, populate the magic link there.
-		if ( false !== strpos( $message, self::MAGIC_LINK_PLACEHOLDER ) ) {
-			$message = str_replace( self::MAGIC_LINK_PLACEHOLDER, $magic_link_url, $message );
-		} else {
-			// Otherwise, append it to the end of the message.
-			$message .= $magic_link_url . "\r\n";
-		}
+		$message .= $magic_link_url . "\r\n";
```

Comment:
Probably for a new PR, but noting that all Reader Activation-related emails should be redesigned and put through this new system.

Reply:
I agree, just didn't want to make this PR too large.